src/main/java/lotto/util/LottoManager.java

```diff
@@ -0,0 +1,143 @@
+package lotto.util;
+
+import static lotto.numbers.Lotto.MAX_LOTTO_NUMBER;
+import static lotto.numbers.Lotto.MIN_LOTTO_NUMBER;
+import static lotto.numbers.Lotto.SIZE_OF_LOTTO;
+import static lotto.util.Money.LOTTO_PRICE;
+
+import camp.nextstep.edu.missionutils.Console;
+import camp.nextstep.edu.missionutils.Randoms;
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lotto.numbers.Lotto;
+import lotto.numbers.UserInputNumbers;
+
+public class LottoManager {
+
+  public static final int PERCENTAGE_FACTOR = 100;
+
+  public void runLotto() {
+    Money money = receivePurchaseAmount();
+    List<Lotto> lottos = generateLottos(money.getAmount());
+    printGeneratedLottoNumbers(lottos);
+    UserInputNumbers receivedLotto = receiveLottoNumber();
+    Map<WinningCheck, Integer> result = calculateWinningResult(lottos, receivedLotto);
+    printWinningResult(result);
+    printProfitRate(money.getAmount(), result);
+  }
+
+  private Money receivePurchaseAmount() {
+    Money money;
+    while (true) {
+      try {
+        System.out.println("구입금액을 입력해 주세요.");
+        String input = Console.readLine();
+        money = new Money(input);
+        break;
+      } catch (IllegalArgumentException e) {
+        System.out.println(e.getMessage());
+      }
+    }
+    return money;
```

Comment:
```suggestion
    while (true) {
            try {
                System.out.println("구입금액을 입력해 주세요.");
                String input = Console.readLine();
                return new Money(input);
            } catch (IllegalArgumentException e) {
                System.out.println(e.getMessage());
            }
        }
```
이렇게 try 문 안에서 직접 반환해버리면 사용하지 않는 변수를 선언할 필요가 없어집니다!

Reply:
네!! 반영하겠습니다! 감사합니다!